pubspec.yaml

```diff
@@ -15,7 +15,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 # In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-version: 1.0.3+4
+version: 1.1.1
```

Comment:
```suggestion
version: 1.1.1+5
```

Reply:
The play store also uses a 'version code' that is an increasing integer. The previous version code was 4, so this increments it to '5'
More info:
https://stackoverflow.com/questions/53570575/flutter-upgrade-the-version-code-for-play-store

Reply:
Good point! I did not have an idea about that rule